extensions/wikia/Recirculation/templates/client/rail.mustache

```diff
@@ -1,13 +1,14 @@
-<section class="recirculation-rail module">
-	<h2>{{ title }}</h2>
+<section class="recirculation-rail module group-{{ group }}">
+	<h2>{{{ title }}}</h2>
```

Comment:
Well, out of all the possibilities not many of them fit nicely to Mustache templating. It's probably fine how it is, I would express more explicitly the title is actually HTML naming it `titleHtml`.